roles.go

```diff
@@ -0,0 +1,183 @@
+package gaia
+
+var (
+	// TODO: Load these in via a file
```

Comment:
Before we merge this PR we should do this or at least move this into the correct subpackage. The first directory level is not the best place for this.